filebeat/README.md

```diff
@@ -190,6 +190,14 @@ namespace which gives it access to the host loopback device, services listening
 on localhost, could be used to snoop on network activity of other pods on the
 same node.
 
+### How do I get multiple beats agents working with hostNetworking enabled?
```

Comment:
This shouldn't be seen as a new change as #997 has been merged. Can you try to rebase master on your branch?